dev/io.openliberty.org.jboss.resteasy.common.ee10/src/org/jboss/resteasy/core/providerfactory/ResteasyProviderFactoryImpl.java

```diff
@@ -0,0 +1,1837 @@
+      return RuntimeType.SERVER;
+   }
+
+   @Override
+   public Collection<String> getPropertyNames()
+   {
+      return getProperties().keySet();
+   }
+
+   @Override
+   public boolean isEnabled(Feature feature)
+   {
+      return getEnabledFeatures().contains(feature);
+   }
+
+   @Override
+   public boolean isEnabled(Class<? extends Feature> featureClass)
+   {
+      Collection<Feature> enabled = getEnabledFeatures();
+      //logger.info("isEnabled(Class): " + featureClass.getName() + " # enabled: " + enabled.size());
+      if (enabled == null)
+         return false;
+      for (Feature feature : enabled)
+      {
+         //logger.info("  looking at: " + feature.getClass());
+         if (featureClass.equals(feature.getClass()))
+         {
+            //logger.info("   found: " + featureClass.getName());
+            return true;
+         }
+      }
+      //logger.info("not enabled class: " + featureClass.getName());
+      return false;
+   }
+
+   @Override
+   public boolean isRegistered(Object component)
+   {
+      return getProviderInstances().contains(component);
+   }
+
+   @Override
+   public boolean isRegistered(Class<?> componentClass)
+   {
+      return getClassContracts().containsKey(componentClass);
+   }
+
+   @Override
+   public Map<Class<?>, Integer> getContracts(Class<?> componentClass)
+   {
+      Map<Class<?>, Integer> classIntegerMap = classContracts.get(componentClass);
+      if (classIntegerMap == null)
+         return Collections.emptyMap();
+      return classIntegerMap;
+   }
+
+   @Override
+   public Set<Class<?>> getClasses()
+   {
+      Set<Class<?>> providerClasses = getProviderClasses();
+      return (providerClasses == null || providerClasses.isEmpty())
+            ? Collections.emptySet()
+            : Collections.unmodifiableSet(providerClasses);
+   }
+
+   @Override
+   public Set<Object> getInstances()
+   {
+      Set<Object> providerInstances = getProviderInstances();
+      return (providerInstances == null || providerInstances.isEmpty())
+            ? Collections.emptySet()
+            : Collections.unmodifiableSet(providerInstances);
+   }
+
+   @Override
+   public Link.Builder createLinkBuilder()
+   {
+      return Utils.createLinkBuilder();
+   }
+
+   @Override
+   public boolean hasProperty(final String name) {
+      return properties.containsKey(name);
+   }
+
+   @Override
+   public SeBootstrap.Configuration.Builder createConfigurationBuilder() {
+      return ResteasySeConfiguration.builder();
+   }
+
+   @Override
+   public CompletionStage<SeBootstrap.Instance> bootstrap(final Application application,
+                                                          final SeBootstrap.Configuration configuration) {                                                      
+      throw new UnsupportedOperationException("WebSphere Liberty does not support the optional Jakarta Rest SE Bootstrap API.");  //Liberty change                                                   
```

Comment:
I would prefer the word WebSphere were not part of this message since it is in Open Liberty repo.  Also I think it should be `Java SE` and not just `SE`

Reply:
Done.